packages/perseus/src/widgets/grapher/grapher.tsx

```diff
@@ -346,10 +346,11 @@ class FunctionGrapher extends React.Component<FunctionGrapherProps> {
     }
 }
 
-type RenderProps = {
-    availableTypes: PerseusGrapherWidgetOptions["availableTypes"];
-    graph: PerseusGrapherWidgetOptions["graph"];
-    plot?: any;
+type RenderProps = Pick<
+    GrapherPublicWidgetOptions,
+    "availableTypes" | "graph"
+> & {
+    plot?: PerseusGrapherWidgetOptions["correct"];
```

Comment:
The types are a little weird here. `correct` gets mapped to `plot` in `staticTransform` but not in `transform`.

Reply:
Would this benefit from the typing check that has been added to some widgets? When I test adding these in, there's an error with the first one, but I don't think it's related to your work 🤔 Relates to the `markings` prop being added as a top-level prop, but it's not added that way in PerseusGrapherWidgetOptions it seems.
```
0 as any as WidgetProps<PerseusGrapherWidgetOptions> satisfies PropsFor<
    typeof Grapher
>;

0 as any as WidgetProps<GrapherPublicWidgetOptions> satisfies PropsFor<
    typeof Grapher
>;
```

Reply:
https://github.com/Khan/perseus/pull/2465